hudi-client/src/test/java/org/apache/hudi/common/HoodieTestDataGenerator.java

```diff
@@ -84,26 +87,35 @@
       + "{\"name\": \"amount\",\"type\": \"double\"},{\"name\": \"currency\", \"type\": \"string\"}]}},";
   public static final String FARE_FLATTENED_SCHEMA = "{\"name\": \"fare\", \"type\": \"double\"},"
       + "{\"name\": \"currency\", \"type\": \"string\"},";
-
   public static final String TRIP_EXAMPLE_SCHEMA =
       TRIP_SCHEMA_PREFIX + FARE_NESTED_SCHEMA + TRIP_SCHEMA_SUFFIX;
   public static final String TRIP_FLATTENED_SCHEMA =
       TRIP_SCHEMA_PREFIX + FARE_FLATTENED_SCHEMA + TRIP_SCHEMA_SUFFIX;
 
+  public static final String TRIP_UBER_SCHEMA = "{\"type\":\"record\",\"name\":\"tripUberRec\",\"fields\":["
```

Comment:
Lets not add the name uber in these example schema as they are generic. Just keep TRIP_SCHEMA and SHORT_TRIP_SCHEMA

Reply:
Sure.